adapters/marsmedia/marsmediatest/supplemental/missing-param.json

```diff
@@ -4,17 +4,11 @@
     "imp": [
       {
         "id": "test-missing-req-param-id",
-        "banner": {
-          "format": [
-            {
-              "w": 300,
-              "h": 250
-            },
-            {
-              "w": 300,
-              "h": 600
-            }
-          ]
+        "video": {
+          "mimes": ["video/mp4"],
+          "protocols": [2, 5],
+          "w": 1024,
+          "h": 576
         },
         "ext": {
           "bidder": {
```

Comment:
Is this change to the marsmedia adapter intentional?

Reply:
Ho yes, it's removed.